go-controller/pkg/libovsdb/libovsdb.go

```diff
@@ -27,7 +27,10 @@ import (
 func newClient(cfg config.OvnAuthConfig, dbModel model.ClientDBModel, stopCh <-chan struct{}) (client.Client, error) {
 	logger := klogr.New()
 	options := []client.Option{
-		client.WithReconnect(types.OVSDBTimeout, &backoff.ZeroBackOff{}),
+		// Reading and parsing the DB after reconnect at scale can (unsurprisingly)
+		// take longer than a normal ovsdb operation. Give it a bit more time so
+		// we don't time out and enter a reconnect loop.
+		client.WithReconnect(types.OVSDBTimeout*2, &backoff.ZeroBackOff{}),
```

Comment:
are you sure 2x is enough? I'd be okey if we went 3 or 5x as well...

Reply:
I've only seen around 10s in scale tests, but sometimes it's enough to overrun the 10s context for the full connect + monitor + parse operation and fail. I figured 2x would be enough?